src/secondaryServer/secondaryserver.py

```diff
@@ -111,14 +115,14 @@ def writeToServer(self, action, *args, **kw):
         out.writeUInt32(0)
         out.writeQString(action)
         
-        for arg in args :            
-            if type(arg) is IntType:
+        for arg in args:
+            if type(arg) is type(int):
```

Comment:
No, put it into GrotheFAF:feature/replays-highlight . This way it get's added to this PR automatically.